packages/web-components/src/popover/popover.template.ts

```diff
@@ -0,0 +1,15 @@
+import { html, ref, slotted, when } from '@microsoft/fast-element';
+import { Popover } from './popover';
+
+export const popoverTemplate = () => html<Popover>`
+  <slot name="anchor" ${slotted('anchorRef')}></slot>
+  ${when(
+    x => x.open,
+    html`
+      <div class="popover-content" ${ref('popoverContentRef')}>
+        ${when(x => x.arrow, html`<div class="popover-arrow" ${ref('arrowRef')}></div>`)}
+        <slot></slot>
+      </div>
+    `,
+  )}
```

Comment:
Thinking a bit more on this, I do wonder if the ergonomic of having the host element represent the "popover" itself is better. If we consider folks needing to associate ARIA, we can apply it directly to the "popover" by setting the attributes which exist on HTMLElement. In the case of the popover being "nested" within the host as it is here, we'll need to enumerate all those attributes and then setup bindings to the internal div to ensure that they get passed accordingly. In the case of any ARIA which requires an ID for association, it won't work because of cross root ARIA. Applying to the host element saves perf cost on the setup/binding (not needed) and allows for associations to be made in the same DOM. Thoughts?